src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -100,9 +101,11 @@ private static Person createEditedPerson(ReadOnlyPerson personToEdit,
         Phone updatedPhone = editPersonDescriptor.getPhone().orElse(personToEdit.getPhone());
         Email updatedEmail = editPersonDescriptor.getEmail().orElse(personToEdit.getEmail());
         Address updatedAddress = editPersonDescriptor.getAddress().orElse(personToEdit.getAddress());
+        Remark updatedRemark = personToEdit.getRemark(); // edit command does not allow editing remarks
```

Comment:
can i ask what is the rationale behind not allowing user to edit remark?

Reply:
cos remark is by itself a more limited editing command and add at the same time

Reply:
i will modify the edit to accomodate remark later, thx for bringing it up